x-pack/legacy/plugins/ml/public/components/navigation_menu/navigation_menu.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, SFC, useState } from 'react';
```

Comment:
nit: `SFC` is deprecated with React 16.7, we should use `FC` instead

Reply:
Fixed in 999248fd7c9196611a3c41fb091ed4d4cc8f8f23